cypress/support/commands.js

```diff
@@ -113,22 +113,7 @@ Cypress.Commands.add('checkA11y', () => {
 
     cy.window({ log: false })
         .then(window => {
-            return window.axe.run(
-                {
-                    include: window.document,
-                    exclude: [['iframe']]
-                },
-                {
-                    rules: {
-                        // @TODO: Review and re-enable this
-                        'color-contrast': { enabled: false },
```

Comment:
I've removed the default excludes in favour of overriding on a per-case basis. We should only be excluding the colour contrast check for known exceptions.